pages/game-page/game.html

```diff
@@ -51,19 +51,30 @@
     </nav>
 
     <!-- Main page content - game page. Philip D.-->
+    <div class="container-fluid hero-container">
```

Comment:
`<div class="container-fluid hero-container">` 
should be 
`<div class="container-fluid game-page-container">`

This would match your css file `.game-page-container` class on line 82 👍

Reply:
thanks @Ian-Garrigan  yeah I had just left it pointing to your background image for the time being as it looked much better 😆

Reply:
will be in next commit